ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/route.tsx

```diff
@@ -27,10 +27,30 @@ const Layout = () => {
       title: 'Overview',
       to: '/organizations/$orgId/products/$productId/repositories/$repoId/runs2/$runIndex',
     },
+    {
+      title: 'Dependencies',
+      to: '/organizations/$orgId/products/$productId/repositories/$repoId/runs2/$runIndex',
+    },
+    {
+      title: 'Vulnerabilities',
+      to: '/organizations/$orgId/products/$productId/repositories/$repoId/runs2/$runIndex',
+    },
+    {
+      title: 'License Findings',
+      to: '/organizations/$orgId/products/$productId/repositories/$repoId/runs2/$runIndex',
+    },
+    {
+      title: 'Rule Violations',
+      to: '/organizations/$orgId/products/$productId/repositories/$repoId/runs2/$runIndex',
+    },
```

Comment:
If non-existing items are added, I'd somehow indicate for the buttons/links that they're currently disabled.